images/mediatypes.go

```diff
@@ -57,6 +57,12 @@ const (
 
 	MediaTypeImageLayerEncrypted     = ocispec.MediaTypeImageLayer + "+encrypted"
 	MediaTypeImageLayerGzipEncrypted = ocispec.MediaTypeImageLayerGzip + "+encrypted"
+
+	// OCI media types
```

Comment:
WASM media types?

Reply:
yes that makes sense 👍, Will update if decided to move forward with adding these types here.